README.md

```diff
@@ -11,6 +11,9 @@ This is the Orders business logic module.
 
 ## Additional information
 
+IMPORTANT NOTE! It is required that user should have permissions `orders-storage.acquisitions-unit-assignments.item.post` to create an purchase order 
```

Comment:
The user needs to have the extra permission, **only** if an **acquisition unit** has to be assigned to the Order, it would be nice to specifically mention that here

Reply:
Done.